src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -566,7 +566,7 @@ export default class Aircraft {
      * @method pushHistory
      */
     pushHistory() {
-        this.history.push([this.position[0], this.position[1]]);
+        this.history.push([this.position.relativePosition[0], this.position.relativePosition[1]]);
```

Comment:
`.relativePosition` getters now made available throughout the file.

Also, for classes that use dynamic positions, left `.position` as a public property, but for all other classes, privatized the position property to `_position`, and added a getter to protect from change.